schema.go

```diff
@@ -83,7 +83,7 @@ func TransToValidSchemeType(typeName string) string {
 		return INTEGER
 	case "uint32", "int32", "rune":
 		return INTEGER
-	case "uint64", "int64":
+	case "uint64", "int64", "big.Int":
```

Comment:
What is the import is something like ?
```go
import integer "math/big"
```

On the other hand big.Int is not a primitive so this change is not valid.